misc/ballerinalang-data-mapper/src/main/java/org/ballerinalang/datamapper/AIDataMapperCodeAction.java

```diff
@@ -105,13 +103,23 @@ public boolean isEnabled(LanguageServerContext serverContext) {
 
                 String uri = context.fileUri();
                 AIDataMapperCodeActionUtil dataMapperUtil = AIDataMapperCodeActionUtil.getInstance();
-                List<TextEdit> fEdits = dataMapperUtil.getAIDataMapperCodeActionEdits(positionDetails, context,
+                ArrayList outputArray = dataMapperUtil.getAIDataMapperCodeActionEdits(positionDetails, context,
```

Comment:
Declaration should be a List

Reply:
Removed this variable. Without an array an object is passed.